### DISCUSSION TITLE
Initial PR points to wrong branch - how to reset?

### DISCUSSION THREAD

**Discussion by r4fterman** [renovatebot/renovate#29797] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Bitbucket and renovatebot 36.212

### Please tell us more about your question or problem

Hi renovate experts,
I am using renovate together with Bitbucket cloud in a self-hosted setup.

When I add a new Bitbucket Cloud repository to the renovate config JSON then renovate picks the branch that is marked as 'main branch' in this Bitbucket reposutiry and not the branch that was specific in the config file.

Once I changed the main branch configuration in Bitbucket I would like to reset renovate and so it picks the correct branch.

What steps do I have to follow in order to get renovate to run on the correct branch?

Thanks, Roman

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
You always need the renovate config to be in the default branch. You use baseBranches to tell Renovate which base branches to target PRs to

**Reply by r4fterman** [reply]
I used "baseBranches": ["myBranch"] but the PR ended up on a different branch. It ended up on the branch that was configured as the "main branch" in this Bitbucket repository. This seems to be an issue especially with Bitbucket Cloud.

I added the renovate.json manually to the desired branch as stated in the intial PR comment:

> RENOVATE IS DISABLED
> Renovate is disabled because there is no Renovate configuration file. To enable Renovate, you can either (a) change this PR's title to get a new onboarding PR, and merge the new onboarding PR, or (b) create a Renovate config file, and commit that file to your base branch.
